moja-kuhinjica-frontend/src/pages/index.tsx

```diff
@@ -1,49 +1,47 @@
-import {ColDiv} from '../styles/global'
-import { Wrapper, SearchContainer, Title, Content, InputWrapper, Input, SearchIcon, SearchButtonDiv, SearchButtonWrapper, SearchButton, ArrowIcon } from '@/styles/style'
+import RestaurantSlider from '@/components/slider/RestaurantSlider'
+import { ColDiv } from '../styles/global'
+import {
+    Wrapper,
+    SearchContainer,
+    Title,
+    Content,
+    InputWrapper,
+    Input,
+    SearchIcon,
+    SearchButtonDiv,
+    SearchButton,
+} from '@/styles/style'
 import Header from '@/components/header/Header'
-import RestaurantList from '@/components/restaurant/restaurant_list/RestaurantList'
 import { RestaurantListWrapper } from '@/styles/style'
 import searchRectangle from '../../public/static/assets/images/search-location.png'
-import searchButton from '../../public/static/assets/images/search-button.png'
-import arrowIcon from '../../public/static/assets/images/arrow.svg'
-import MySlider from '@/components/slider/MySlider'
-
 
 const Home = () => {
     return (
-      <ColDiv>
-          <Wrapper height="887px" >
-              <Header />
-              <SearchContainer justifyContent="center" width="50%">
-                  <Title>moja klopica</Title>
-                  <Content>
-                      Lorem ipsum dolor sit amet, consectetur adipiscing elit.
-                  </Content>
-                  <InputWrapper
-                      height="55px"
-                      width="480px"
-                      alignItems="center"
-                  >
-                      <Input
-                          placeholder="Unesi adresu na kojoj se nalaziš"
-                          type="text"
-                      />
-                      <SearchIcon src={searchRectangle} alt="" />
-                      <SearchButtonDiv>
-                          <SearchButtonWrapper>
-                              <SearchButton src={searchButton} alt="" />
-                              <ArrowIcon src={arrowIcon} alt="" />
-                          </SearchButtonWrapper>
-                      </SearchButtonDiv>
-                  </InputWrapper>
-              </SearchContainer>
-          </Wrapper>
-          <MySlider/>
-          {/* <RestaurantListWrapper>
-              <RestaurantList />
-          </RestaurantListWrapper> */}
-      </ColDiv>
-  )
+        <ColDiv>
+            <Wrapper height="840px">
+                <Header />
+                <SearchContainer justifyContent="center" width="50%">
+                    <Title>moja klopica</Title>
+                    <Content>
+                        Lorem ipsum dolor sit amet, consectetuer adipiscing.
+                    </Content>
+                    <InputWrapper>
+                        <Input
```

Comment:
When I type longer string text goes above send button in input. Please fix this
![Screenshot 2023-01-18 at 15 25 43](https://user-images.githubusercontent.com/85338076/213198428-d8ecf5f0-15a5-41e9-91d3-3a6f51dfa7b2.png)

Reply:
Good catch, done.